src/deckgl-layers/h3-hexagon-cell-layer/h3-hexagon-cell-layer.js

```diff
@@ -40,17 +67,16 @@ export default class H3HexagonCellLayer extends HexagonCellLayer {
       nvertical: 1
     });
 
-    const pos = cylinderGeometry.attributes.positions.value;
-
-    if (transform) {
-      const adjusted = transformCylinderPositions(pos, transform);
+    if (distortion) {
+      const pos = cylinderGeometry.attributes.positions.value;
+      const adjusted = distortCylinderPositions(pos, distortion);
       cylinderGeometry.attributes.positions.value = adjusted;
     }
 
     return cylinderGeometry;
   }
 
-  getTransform() {
+  getDistortion() {
```

Comment:
The name of this function implies it's a pure getter, but it's setting state. Should probably change the name to e.g. `calcDistortion` or even just `setDistortion`.